src/main/java/no/sikt/nva/UnisContent.java

```diff
@@ -0,0 +1,163 @@
+package no.sikt.nva;
+
+import static no.sikt.nva.UnisContentScrapingResult.VALID_HEADERS;
+import java.util.Date;
+import no.sikt.nva.brage.migration.common.model.record.PublisherAuthorityEnum;
+import no.sikt.nva.brage.migration.common.model.record.Record;
+import no.sikt.nva.brage.migration.common.model.record.license.BrageLicense;
+import no.sikt.nva.exceptions.ExcelException;
+import nva.commons.core.StringUtils;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.DateUtil;
+import org.apache.poi.ss.usermodel.Row;
+
+public final class UnisContent {
+    public static final int FIRST_COLUMN = 0;
+    public static final int CRISTIN_ID_COLUMN = 0;
+    public static final int TITLE_COLUMN = 1;
+    public static final int PUBLISHER_AUTHORITY_COLUMN = 2;
+    public static final int EMBARGO_COLUMN = 3;
+    public static final int LICENCE_COLUMN = 4;
+    public static final int FILENAME_COLUMN = 5;
+    public static final String ERROR_MESSAGE_INVALID_NUMBER_OF_COLUMNS = "Invalid number of columns";
+    public static final String ERROR_MESSAGE_MISSING_FIRST_COLUMN_VALUE = "Missing first column value";
+    public static final String ERROR_MESSAGE_MISSING_CRISTIN_ID = "Missing Cristin Id";
+    public static final String ERROR_MESSAGE_INVALID_CRISTIN_ID = "Invalid Cristin Id";
+    public static final String ERROR_MESSAGE_MISSING_TITLE = "Missing Title";
+    public static final String ERROR_MESSAGE_MISSING_PUBLISHER_AUTHORITY = "Missing Publisher Authority";
+    public static final String ERROR_MESSAGE_INVALID_PUBLISHER_AUTHORITY = "Invalid Publisher Authority value";
+    public static final String ERROR_MESSAGE_INVALID_EMBARGO_FORMAT = "Invalid Embargo format";
+    public static final String ERROR_MESSAGE_MISSING_LICENCE = "Missing Licence";
+    public static final String ERROR_MESSAGE_INVALID_LICENCE = "Invalid Licence value";
+    public static final String ERROR_MESSAGE_MISSING_FILENAME = "Missing Filename";
+
+    private int cristinId;
+    private String title;
+    private PublisherAuthorityEnum publisherAuthority;
+    private Date embargo;
+    private BrageLicense licence;
+    private String filename;
```

Comment:
Will these properties ever change once the object is created?
I don't think so. Make them final.